src/main/webapp/app/course/learning-paths/participate/lecture-unit/learning-path-lecture-unit-view.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, Input } from '@angular/core';
+import { HttpErrorResponse } from '@angular/common/http';
+import { LectureUnit, LectureUnitType } from 'app/entities/lecture-unit/lectureUnit.model';
+import { onError } from 'app/shared/util/global.utils';
+import { Lecture } from 'app/entities/lecture.model';
+import { LectureUnitService } from 'app/lecture/lecture-unit/lecture-unit-management/lectureUnit.service';
+import { AlertService } from 'app/core/util/alert.service';
+import { isCommunicationEnabled, isMessagingEnabled } from 'app/entities/course.model';
+import { DiscussionSectionComponent } from 'app/overview/discussion-section/discussion-section.component';
+
+export interface LectureUnitCompletionEvent {
+    lectureUnit: LectureUnit;
+    completed: boolean;
+}
+
+@Component({
+    selector: 'jhi-learning-path-lecture-unit-view',
+    templateUrl: './learning-path-lecture-unit-view.component.html',
+})
+export class LearningPathLectureUnitViewComponent {
+    @Input() lecture: Lecture;
+    @Input() lectureUnit: LectureUnit;
+    readonly LectureUnitType = LectureUnitType;
+
+    discussionComponent?: DiscussionSectionComponent;
+
+    constructor(private lectureUnitService: LectureUnitService, private alertService: AlertService) {}
```

Comment:
codacy

Reply:
If I change this, our ESLint script complains.